utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -810,7 +810,7 @@ where
 }
 
 /// Build all extensions that we typically use.
-pub(crate) fn full_extensions() -> Extensions {
+pub(crate) fn full_extensions<H: HostFunctions>(wasm_executor: WasmExecutor<H>) -> Extensions {
```

Comment:
Considering the bounds on `ReadRuntimeVersionExt::new` maybe this would be better?

```suggestion
pub(crate) fn full_extensions(read_runtime_version: impl ReadRuntimeVersionExt + 'static) -> Extensions {
```

Probably doesn't matter though; feel free to ignore.